devicetypes/bspranger/xiaomi-temperature-humidity-sensor.src/xiaomi-temperature-humidity-sensor.groovy

```diff
@@ -134,7 +140,14 @@ metadata {
 def parse(String description) {
 
     // send event for heartbeat
-    def now = new Date().format("yyyy MMM dd EEE h:mm:ss a", location.timeZone)
+    def now
+	if(dateformat == "US" || dateformat == "" || dateformat == null)
```

Comment:
I noticed that Brian had to change references of the variable `tempOffset` to `settings.tempOffset` in order to get conditionals / boolean operands to work correctly in the `parseTemperature()` function.

I haven't wrapped my head around global variables in Groovy, but doesn't that mean all references to `dateformat`, `voltsmax`, and `voltsmin` should be changed to `settings.dateformat`, `settings.voltsmax`, and `settings.voltsmin`?